libs/designsystem/src/lib/kirby.module.ts

```diff
@@ -138,6 +137,7 @@ const exportedDeclarations = [
   AccordionItemComponent,
   RadioComponent,
   RadioGroupComponent,
+  SizeDirective,
```

Comment:
Don't think we need to include it in KirbyModule - it just needs to be exported in the public API/namespace `@kirbydesign/designsystem`
```suggestion
```

Reply:
Like this 9aca432?